Tests/Packet++Test/main.cpp

```diff
@@ -250,8 +250,12 @@ int main(int argc, char* argv[])
 	PTF_RUN_TEST(NtpParsingV4Tests, "ntp");
 	PTF_RUN_TEST(NtpCraftingTests, "ntp");
 
-	PTF_RUN_TEST(TelnetCommandParsingTests, "telnet");
+  	PTF_RUN_TEST(TelnetCommandParsingTests, "telnet");
 	PTF_RUN_TEST(TelnetDataParsingTests, "telnet");
 
+	PTF_RUN_TEST(FtpParsingIpv4Tests, "ftp");
+	PTF_RUN_TEST(FtpParsingIpv6Tests, "ftp");
+	PTF_RUN_TEST(FtpCraftingTests, "ftp");

```

Comment:
Can we add another test for updating an existing FTP request/response?

Reply:
@egecetin did you add a test to update an existing layer (`FtpEditTest`)

Reply:
@seladb I added lines to `FtpCraftingTests`,

https://github.com/egecetin/PcapPlusPlus/blob/c841c2d7370eac399d708a2690e4d74355ec7a0e/Tests/Packet%2B%2BTest/Tests/FtpTests.cpp#L149-L176